run_scripts/checkout_rapid_pro_tools.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -ne 1 ]; then
+    echo "Usage: ./checkout_rapid_pro_tools.sh <rapid-pro-tools-dir>"
+    echo "Ensures that a copy of the RapidProTools project exists in  'rapid-pro-tools-dir' by cloning/fetching as "
+    echo "necessary, and checking-out the version needed by this project"
+    exit
+fi
+
+RAPID_PRO_TOOLS_DIR="$1"
+
+RAPID_PRO_TOOLS_REPO="https://github.com/AfricasVoices/RapidProTools.git"
+TAG="v0.2.4"
```

Comment:
I would use the (recently released) v0.2.5 (https://github.com/AfricasVoices/RapidProTools/releases/tag/v0.2.5).

Very little has changed, but you may as well enjoy the benefit of better error messages.